crp_extract.py

```diff
@@ -140,4 +157,6 @@ def main():
     with open(output_path + "metadata.json", "w") as f:
```

Comment:
Because `output_path` was changed, `metadata.json` is now being stored in the current directory as a file called `output_pathmetadata.json`. I think to fix it is as simple as:

```
with open(os.path.join(output_path, 'metadata.json'), 'w') as f:
```

Reply:
yep I missed that one.